src/debugtopus.ts

```diff
@@ -76,18 +77,53 @@ export const prepareTestRun = async ({
   if (!existsSync(tempDir)) {
     await fs.mkdir(tempDir);
   }
-
+  const fileNameUUID = randomUUID();
   const outputDir = path.join(tempDir, "output");
 
-  const testFilePath = path.join(tempDir, `${randomUUID()}.spec.ts`);
+  const testFilePath = path.join(tempDir, `${fileNameUUID}.spec.ts`);
   writeFileSync(testFilePath, code);
 
-  const configFilePath = path.join(tempDir, `${randomUUID()}.config.ts`);
+  const configFilePath = path.join(tempDir, `${fileNameUUID}.config.ts`);
   writeFileSync(configFilePath, getConfig(url, outputDir));
 
   return { testFilePath, configFilePath, outputDir, packageRootDir };
 };
 
+async function ensureChromiumIsInstalled(packageRootDir: string) {
```

Comment:
arrow function please and return type 😬

Reply:
and some tests please

Reply:
and adjust the docs in https://github.com/OctoMind-dev/mintlify-docs as well so it doesn't have the "error" anymore ;)